wiki/People/Beatmap_Nomination_Group/en.md

```diff
@@ -1,11 +1,14 @@
-The **Beatmap Nomination Group**, more commonly referred to as BN, is a group in charge of [beatmap nomination](Ranked_Beatmap "wikilink"). This group has undergone many changes recently. As of August 22, 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums. The most recent change was on February 1, 2015 when the team was changed from Beatmap Appreciation Team to Beatmap Nominator Group and lost their red forum name.
+Beatmap Nomination Group
+=========================
+
+The **Beatmap Nomination Group**, more commonly referred to as BN, is a group in charge of [beatmap nomination](/wiki/Beatmaps). This group has undergone many changes recently. As of August 22, 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums. The most recent change was on February 1, 2015 when the team was changed from Beatmap Appreciation Team to Beatmap Nominator Group and lost their red forum name.
 
 Roles and Responsibilities
 --------------------------
 
-![The Nominator Badge](Nominator.png "fig:The Nominator Badge")
+![The Nominator Badge](Nominator.png "The Nominator Badge")
 
-Members of the **Beatmap Nomination Group** are responsible primarily for [modding](Modding "wikilink") the [pending beatmaps](Pending_Beatmap "wikilink") into the ranking cycle. BN are the people to go to for any modding or advice requests about anything mapping or modding related.
+Members of the **Beatmap Nomination Group** are responsible primarily for [modding](/wiki/Modding) the [pending beatmaps](/wiki/Beatmaps) into the ranking cycle. BN are the people to go to for any modding or advice requests about anything mapping or modding related.
 
 Becoming a BN
 -------------
```

Comment:
- line 23: it speaks about the modes column that was on the old wiki.. this could be reworded to note that the game modes are the ones said person focuses at, but aren't forced to nominate just those maps.
- line 25: update game mode names